README.md

```diff
@@ -48,11 +48,12 @@ Usage
 Add the following to your `init.el` (after calling `package-initialize`):
 
 ```el
-(when (memq window-system '(mac ns))
+(when (memq window-system '(mac ns x))
   (exec-path-from-shell-initialize))
```

Comment:
Many desktop environments run from display managers and don't run inside the standard shell session. I'm running Gnome 3 from gdm and if I check PATH without first running `exec-path-from-shell-initialize`, it's not the correct value.